cmd/infracost/testdata/breakdown_config_file/breakdown_config_file.golden

```diff
@@ -17,7 +17,7 @@
       "name": "infracost/infracost/cmd/infracost/testdata/breakdown_config_file/dev",
       "displayName": "testdata-breakdown_config_file-dev",
       "metadata": {
-        "path": "REPLACED_PROJECT_PATH/infracost/infracost/cmd/infracost/testdata/breakdown_config_file/dev",
+        "path": "/Users/liamg/src/infracost/cmd/infracost/testdata/breakdown_config_file/dev",
```

Comment:
hmm seems something has gone wrong here - this should have the `REPLACED_PROJECT_PATH` variable